frontend/src/lib/utils/getErrorMessage.ts

```diff
@@ -0,0 +1,7 @@
+export const getErrorMessage = (err: unknown): string => {
+  if (err instanceof Error) {
+    return err.message;
+  }
+
+  return JSON.stringify(err);
+};
```

Comment:
Ok then, I've renamed the function to `extractErrorMsg` for more clarity, and adjusted handling so that in case it couldn't extract the message it would  attempt to`JSON.stringify` it.